### PR TITLE
RR-303 make the view note link black on print

### PR DESCRIPTION
**Pull request by chrimesdev**
The “View note” link should be black so that it doesn’t use colour ink, usually we would hide links all together as they're not clickable on paper. But I think in this instance it needs the View note as a heading for context.

Screenshot 2023-09-13 at 18 09 25

### CODE REVIEW

File: assets/scss/print.scss
Comment (nathanrussell-moj-digital) [inline assets/scss/print.scss:10]: LGTM; out of interest, could you use the variable`$govuk-text-colour` instead of the hex reference?
https://design-system.service.gov.uk/styles/colour/

**Review by nathanrussell-moj-digital — DISMISSED** [review]
1 very minor comment/question; but otherwise LGTM 👍

**Review by mattwills-moj-digital — APPROVED** [review]

File: assets/scss/print.scss
Comment (chrimesdev) [reply assets/scss/print.scss]: We didn't previously have access to the colour variables in the `print.scss`, I've included the relevant imports we need from `govuk` (not all components styles just the base/core) to allow us to use variables here.

**Comment by chrimesdev** [comment]
This will need re-approval (if happy) as I pushed to the branch and its dismissed reviews.

**Review by nathanrussell-moj-digital — APPROVED** [review]
LGTM 👍

**Pull request merged**
